### PR TITLE
Removed uses of non-existing parameter in annotation

### LINKED ISSUE #1697: Humidifier_u.mo uses use_T_in which does not exist
The model `Humidifier_u` uses the parameter `use_T_in` in the graphical annotation, but this parameter does not exist in this model.

### PR DESCRIPTION
**Pull request by mwetter**
This closes #1697

### CODE REVIEW

**Comment by mwetter** [comment]
@LauraMaier : can you please review this so we can go ahead and merge it. Thanks.

File: IBPSA/Fluid/Humidifiers/Humidifier_u.mo [hunk 1]
```diff
@@ -107,6 +100,12 @@ equation
               {42,-28},{42,-26}},
           lineColor={255,255,255},
           fillColor={255,255,255},
+          fillPattern=FillPattern.Solid),
```
Comment (LauraMaier) [inline IBPSA/Fluid/Humidifiers/Humidifier_u.mo:103]: Was this change intended?

**Review by LauraMaier — APPROVED** [review]
Just a small question that might be unnecessary for the git history. Other than that: all fine

File: IBPSA/Fluid/Humidifiers/Humidifier_u.mo (hunk 1 above)
Comment (mwetter) [reply IBPSA/Fluid/Humidifiers/Humidifier_u.mo:103]: Yes, this change is good and consistent with the other drawing elements in this icon.
